src/crypto-api.ts

```diff
@@ -77,6 +77,10 @@ export interface CryptoApi {
     /**
      * Get the device information for the given list of users.
      *
+     * First of all, the devices are retrieved from the tracked users in the local client
+     * If the users are not tracked and the `downloadUncached` parameter is set at `true`,
+     * A `/keys/query` request is made to the local user homeserver to retrieve these devices
```

Comment:
This assumes the user knows what a "tracked user" is, which is an internal detail.

```suggestion
     * For any users whose device lists are cached (due to sharing an encrypted room with the user), the
     * cached device data is returned.
     *
     * If there are uncached users,  and the `downloadUncached` parameter is set to `true`,
     * a `/keys/query` request is made to the server to retrieve these devices.
```